src/sentry/processing/realtime_metrics/base.py

```diff
@@ -43,22 +93,42 @@ def get_lpq_projects(self) -> Set[int]:
         """
         pass
 
-    def add_project_to_lpq(self, project_id: int) -> None:
+    def add_project_to_lpq(self, project_id: int) -> bool:
         """
         Moves a project to the low priority queue.
 
         This forces all symbolication events triggered by the specified project to be redirected to
         the low priority queue, unless the project is manually excluded from the low priority queue
         via the `store.symbolicate-event-lpq-never` kill switch.
+
+        Returns True if the project was a new addition to the list. Returns False if it was already
+        assigned to the low priority queue.
         """
         pass
 
-    def remove_projects_from_lpq(self, project_ids: Set[int]) -> None:
+    def remove_project_from_lpq(self, project_id: int) -> bool:
+        """
+        Removes a project from the low priority queue.
+
+        This restores the specified project back to the regular queue, unless it has been
+        manually forced into the low priority queue via the `store.symbolicate-event-lpq-always`
+        kill switch.
+
+        Returns True if the project was assigned to the queue prior to its removal. Returns False if
+        it wasn't assigned to the queue to begin with.
```

Comment:
Similar to above: what happens if the project was in the queue, but hasn't been removed because it's in the always list?

Reply:
let me try to address both questions in one comment:

i think i phrased the documentation poorly, particularly when it comes to mentioning these two kill switches. any mutations of the value stored in redis (ie `store.symbolicate-event-lpq-selected`) do not read or take into consideration the two manual kill switches (`store.symbolicate-event-lpq-never` and `store.symbolicate-event-lpq-always`). 

i mentioned the switches as a way to provide context if a project's events continued to be routed into the "wrong" queue despite these clearly being invoked, but i think that just made things more confusing. the only place that does collect all three of `-selected`, `-never`, and `-always` exclusively reads the contents of those values, but they do not mutate them.

i've changed the wording on the docstring to better reflect this in this PR and the parent PR: https://github.com/getsentry/sentry/pull/28757

let me know what you think of the updated docstrings.

Reply:
Ok, let me see if I understand this correctly. Before, my main issue with the documentation was this: does the return value of (e.g.) `add_project_to_lpq` reflect whether the project was added or whether it is now in the lpq? But if the killswitches aren't taken into account, the latter is meaningless because an addition always results in the project being in the lpq. Thus returning whether it was added (or removed) is the only useful option.